ckanext/unhcr/actions.py

```diff
@@ -853,7 +859,17 @@ def access_request_list_for_user(context, data_dict):
             ),
             and_(
                 access_requests_table.c.object_type == "user",
+                or_(
+                    not_(access_requests_table.c.data.has_key("user_request_type")),
+                    access_requests_table.c.data["user_request_type"].astext == USER_REQUEST_TYPE_NEW,
+                ),
                 access_requests_table.c.data["default_containers"].has_any(array(containers)),
+            ),
+            and_(
+                access_requests_table.c.object_type == "user",
+                access_requests_table.c.data["user_request_type"].astext == USER_REQUEST_TYPE_RENEWAL,
+                access_requests_table.c.data.has_key("users_who_can_approve"),
+                access_requests_table.c.data["users_who_can_approve"].contains([user_id]),
```

Comment:
```suggestion
                access_requests_table.c.data["users_who_can_approve"].contains([user.id]),
```

This wasn't working when I tested locally because we invoke `access_request_list_for_user` with `'user': toolkit.c.user` which makes `user_id` the user _name_, not the id. Switching to `user.id` fixes it.